### PR TITLE
Fix documentation formating

### PR DESCRIPTION
**Pull request by JanuszL**
- fixes css for the most recent version of sphinx_rtd_themes:
  sidebar section header formating
- escape special rst character in mel filter bank operator to fix
  warning

Signed-off-by: Janusz Lisiecki <jlisiecki@nvidia.com>

## Description
- [ ] **Bug fix** (*non-breaking change which fixes an issue*)
- [ ] **New feature** (*non-breaking change which adds functionality*)
- [ ] **Breaking change** (*fix or feature that would cause existing functionality to not work as expected*)
- [ ] **Refactoring** (*Redesign of existing code that doesn't affect functionality*)
- [x] **Other** (*e.g. Documentation, Tests, Configuration*)

### What happened in this PR
- fixes css for the most recent version of sphinx_rtd_themes:
  sidebar section header formating
- escape special rst character in mel filter bank operator to fix
  warning

#### Additional information
- Affected modules and functionalities:
  - mel_filter_bank.cc
  -  layout.html

- Key points relevant for the review:
  - NA

### Tests
- [ ] Existing tests apply
- [ ] New tests added
  - [ ] Python tests
  - [ ] GTests
  - [ ] Benchmark
  - [ ] Other
- [x] N/A

### Documentation
- [ ] Existing documentation applies
- [ ] Documentation updated
  - [ ] Docstring
  - [ ] Doxygen
  - [ ] RST
  - [ ] Jupyter
  - [x] Other
- [ ] N/A

#### Requirements
- [ ] Implements new requirements
- [ ] Affects existing requirements
- [x] N/A

**REQ IDs**: N/A

**JIRA TASK**: N/A

### CODE REVIEW

**Comment by JanuszL** [comment]
!build

**Review by awolant — APPROVED** [review]

File: dali/operators/audio/mel_scale/mel_filter_bank.cc [hunk 1]
```diff
@@ -24,7 +24,7 @@ DALI_SCHEMA(MelFilterBank)
 triangular filters.
 
 The frequency ('f') dimension is selected from the input layout.
-In case of no layout, "f", "ft", or "*ft" is assumed, depending on the number of dimensions.
+In case of no layout, "f", "ft", or "\*ft" is assumed, depending on the number of dimensions.
```
Comment (szalpal) [inline dali/operators/audio/mel_scale/mel_filter_bank.cc:27]: If we can afford to run the CI once again, there's a typo a little bit below:

If this value is not provided, ``sample_rate /2`` is used.

to

If this value is not provided, ``sample_rate/2`` is used.

**Review by szalpal — APPROVED** [review]

File: dali/operators/audio/mel_scale/mel_filter_bank.cc (hunk 1 above)
Comment (JanuszL) [reply dali/operators/audio/mel_scale/mel_filter_bank.cc:27]: Done

**Comment by JanuszL** [comment]
!build

**Pull request merged**